.github/workflows/php-lint.yml

```diff
@@ -52,7 +52,9 @@ jobs:
         run: composer validate --strict
       - name: Composer Install
         run: composer install --no-interaction --no-progress
+      - name: Install `npm-run-all` utility
+        run: npm install npm-run-all -g
```

Comment:
Why would this need to be installed globally if it's already installed locally?

Reply:
We aren't installing NPM deps in the PHP Lint workflow, and installing this utility globally took ~2 seconds while `npm install` takes ~20 seconds.